plasmapy/atomic/nuclear.py

```diff
@@ -32,28 +33,28 @@ def nuclear_binding_energy(argument, mass_numb=None):
 
     Examples
     --------
-    >>> nuclear_binding_energy('Fe-56')
-    <Quantity 7.674003137600576e-11 J>
+    >>> from astropy import units as u
+    >>> nuclear_binding_energy('Fe-56').to(u.MeV)
+    <Quantity 492.25957875689096 MeV>
     >>> nuclear_binding_energy(26, 56)
-    <Quantity 7.674003137600576e-11 J>
+    <Quantity 7.88686788449147e-11 J>
```

Comment:
When calculating the binding energy before, I used the mass of the isotope rather than the mass of the nucleus.  That difference is due to the mass of the electrons.

Reply:
Ah, I see. That's cool, then :)